Userland/Libraries/LibX86/Instruction.h

```diff
@@ -518,8 +525,12 @@ class MemoryOrRegisterReference {
         u16 m_displacement16;
     };
 
-    u8 m_rm_byte { 0 };
-    u8 m_sib { 0 };
+    u8 m_mod : 2 { 0 };
+    u8 m_reg : 4 { 0 };
+    u8 m_rm : 4 { 0 };
```

Comment:
At this point it should technically still be 3 each and 4 only with the next commit,
But you don't need to change it, if you don't want

Reply:
(Likely same for SIB, but haven't read that part of the manual)